operator/flags.go

```diff
@@ -32,12 +32,15 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	// AWS dedicated flags
-	flags.Int(option.AWSClientBurst, defaults.AWSClientBurst, "Burst value allowed for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientBurst)
-
-	flags.Float64(option.AWSClientQPSLimit, defaults.AWSClientQPSLimit, "Queries per second limit for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientQPSLimit)
+	flags.Int(option.AWSClientBurstDeprecated, defaults.IPAMAPIBurst, "")
+	flags.MarkDeprecated(option.AWSClientBurstDeprecated, fmt.Sprintf("please use --%s", option.IPAMAPIBurst))
+	flags.Int(option.IPAMAPIBurst, defaults.IPAMAPIBurst, "Upper burst limit when accessing external APIs")
+	option.BindEnv(option.IPAMAPIBurst)
+
+	flags.Float64(option.AWSClientQPSLimitDeprecated, defaults.IPAMAPIQPSLimit, "")
+	flags.MarkDeprecated(option.AWSClientQPSLimitDeprecated, fmt.Sprintf("please use --%s", option.IPAMAPIQPSLimit))
```

Comment:
`option.BindEnv(option.AWSClientQPSLimit)` seems to have been accidentally removed. Also, please add a new line between lines 41 and 42 for better readability.

Reply:
https://github.com/cilium/cilium/pull/10089#discussion_r385744697